hack/testing/rsyslog/README.md

```diff
@@ -0,0 +1,64 @@
+# Guidelines for Using Ansible Roles
+
+## Sample Usage of the Rsyslog Ansible Roles
+
+Command line
+```
+ansible-playbook [-vvv] -e@vars.yaml --become --become-user root --connection local -i inventory_file playbook.yaml
+```
+Sample inventory file for the es-ops enabled case
+```
+[masters]
+localhost ansible_ssh_user=${RSYSLOG_ANSIBLE_SSH_USER} openshift_logging_use_ops=True
+
+[nodes]
+localhost ansible_ssh_user=${RSYSLOG_ANSIBLE_SSH_USER} openshift_logging_use_ops=True
+```
+Sample vars.yaml to deploy viaq scripts 
+```
+rsyslog__enabled: true
+# install viaq packages & config files
+rsyslog__viaq: true
+rsyslog__group: root
+rsyslog__user: root
+rsyslog__capabilities: viaq
+```
+The following configuration files are deployed in /etc by the set of variables.
```

Comment:
I'm probably missing something, but where are the configuration files that should be deployed actually get deployed?

Reply:
I'm in the middle of rewriting README.md.  
Please give me some time for reviewing it...  Thanks.